packages/webcomponents/src/ui-components/form/iframe-input.tsx

```diff
@@ -74,30 +82,37 @@ export class IframeInput {
     }
   }
 
-  private get fontFamily() {
-    return btoa(getComputedStyle(document.body).fontFamily);
-  }
-
   private get part() {
     return `input form-control-text ${this.isFocused ? 'input-focused' : ''} ${!this.isValid ? 'input-invalid' : ''}`;
   }
 
+  private setFontFamily() {
+    this.hiddenInput = this.el.querySelector('input');
```

Comment:
Is the query selector necessary since the input has a ref on it?

Reply:
good catch, there's no point on doing that.